examples/cluster/nodejs/index.ts

```diff
@@ -81,11 +81,17 @@ const cluster = new rke.Cluster(`actions`, {
     cloudProvider: {
         name: "aws"
     },
+    // Some combination of the three settings below appears to be required to unblock tests.
+    // This is likely related to https://github.com/rancher/terraform-provider-rke/issues/370 and/or
+    // https://github.com/rancher/terraform-provider-rke/issues/404 from the upstream provider.
+    delayOnCreation: 180,
+    disablePortCheck: true,
+    enableCriDockerd: true,
```

Comment:
Ah, you beat me to it - was attempting to get this running on the side as well! I believe we actually only need `enableCriDockerd: true`. In the older RKE provider, the default K8s version used is <v1.24. The newer upstream provider that we were trying to bring in uses upstream RKE that targets K8s v1.24+ by default. v1.24 deprecated and removed the docker shim, which is why our tests fails without this flag

Should we remove `delayOnCreation: 180, disablePortCheck: true`? I don't believe these are necessary to unblock tests unless you think otherwise.

Reply:
Ahh - great. Yeah - that makes sense.  I’ll remove them. I suspect it is a bug in the upstream terraform provider that this is required.

Reply:
The upstream provider uses k8s v1.24+ as the default Kubernetes version, but the default flag for `enableCriDockerd` is `false`. We might need to flip the default `enableCriDockerd` to be `true` in our provider so users do not need to explicitly set `enableCriDockerd` for a default RKE cluster?

Reply:
I believe this is a bug in the upstream provider, not in this repo.  We should report it upstream and get it fixed there.  My assumption is that it is likely the underlying cause of https://github.com/rancher/terraform-provider-rke/issues/404 and/or https://github.com/rancher/terraform-provider-rke/issues/370 as well.